frequency-plans.yml

```diff
@@ -2,104 +2,121 @@
   name: Europe 863-870 MHz
   description: Default frequency plan for Europe
   base-frequency: 868
+  country-codes: [al, ad, ao, at, bh, be, ba, bw, bg, cg, hr, cy, cz, dk, ee, sz, fi, fr, gr, hu, is, ie, it, lv, ls, li, lt, lu, mg, mw, mt, mu, md, me, mz, na, nl, mk, ph, pl, pt, ro, ru, sa, rs, sc, sk, si, za, es, se, ch, tz, tr, ae, gb, va, zm, zw]
   file: EU_863_870.yml
 
 - id: EU_863_870_TTN
   name: Europe 863-870 MHz (TTN)
   description: TTN Community Network frequency plan for Europe, using SF9 for Rx2
   base-frequency: 868
   base-id: EU_863_870
+  country-codes: [al, ad, ao, at, bh, be, ba, bw, bg, cg, hr, cy, cz, dk, ee, sz, fi, fr, gr, hu, is, ie, it, lv, ls, li, lt, lu, mg, mw, mt, mu, md, me, mz, na, nl, mk, ph, pl, pt, ro, ru, sa, rs, sc, sk, si, za, es, se, ch, tz, tr, ae, gb, va, zm, zw]
   file: EU_863_870_TTN.yml
 
 - id: US_902_928_FSB_1
   name: United States 902-928 MHz, FSB 1
   description: Default frequency plan for the United States and Canada, using sub-band 1
   base-frequency: 915
+  country-codes: [bo, ca, co, cr, do, ec, gy, mx, pa, py, pe, pr, sr, us, uy, ve]
   file: US_902_928_FSB_1.yml
 
 - id: US_902_928_FSB_2
   name: United States 902-928 MHz, FSB 2 (TTN)
   description: TTN Community Network frequency plan for the United States and Canada, using sub-band 2
   base-frequency: 915
+  country-codes: [[bo, ca, co, cr, do, ec, gy, mx, pa, py, pe, pr, sr, us, uy, ve]
   file: US_902_928_FSB_2.yml
 
 - id: AU_915_928_FSB_1
   name: Australia 915-928 MHz, FSB 1
   description: Default frequency plan for Australia, using sub-band 1
   base-frequency: 915
+  country-codes: [br, cl, nz, ar, au]
   file: AU_915_928_FSB_1.yml
 
 - id: AU_915_928_FSB_2
   name: Australia 915-928 MHz, FSB 2 (TTN)
   description: TTN Community Network frequency plan for Australia, using sub-band 2
   base-frequency: 915
+  country-codes: [br, cl, nz, ar, au]
   file: AU_915_928_FSB_2.yml
 
 - id: AU_915_928_FSB_6
   name: Australia 915-928 MHz, FSB 6
   description: Frequency plan for Australia, using sub-band 6, which overlaps with Asia 923-925 MHz
   base-frequency: 915
+  country-codes: [br, cl, nz, ar, au]
   file: AU_915_928_FSB_6.yml
 
 - id: CN_470_510_FSB_11
   name: China 470-510 MHz, FSB 11
   description: TTN Community Network frequency plan for China, using sub-band 11
   base-frequency: 470
+  country-codes: [cn]
   file: CN_470_510_FSB_11.yml
 
 - id: AS_920_923
   name: Asia 920-923 MHz
   description: TTN Community Network frequency plan for Asian countries, using frequencies ≤ 923 MHz
   base-frequency: 915
+  country-codes: [jp, my, sg]
```

Comment:
We need to check which countries require LBT

Reply:
Yep, but how? Google turns up almost nothing, this is all I found. https://www.murata.com/en-eu/support/faqs/products/lpwa/lora/software/0006

The regulatory documents have no hits for LBT, maybe they call it something else? https://www.arib.or.jp/english/html/overview/doc/5-STD-T108v1_0-E1.pdf

Reply:
> We need to check which countries require LBT

japan and korea according to https://lora-alliance.org/sites/default/files/2020-01/rp_2-1.0.0_final_release.pdf